src/screens/Home.tsx

```diff
@@ -107,9 +108,9 @@ export default function Home(): ReactElement {
           action: () =>
             dispatch({
               type: DispatchAction.SORT,
-              payload: { sort: Sort.Rating },
```

Comment:
This is because we don't have an upvote/downvote functionality right?

Reply:
Before we planned to do rating which is a property we could only get through the google places API -- and so to sort by rating we need to make an API call for each promotion which would be costly so I think we decided to do this instead